.claude/agents/agency/planner.md

```
@@ -0,0 +1,48 @@
+permissionMode: default
+maxTurns: 100
+memory: project
+skills:
+  - agency-client-interview
+---
+
+# Planner - Agency Project Strategist
+
+## FROZEN ZONE
+
+### Identity
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)